cloudfront/invalidation/lambda/.gitignore

```diff
@@ -0,0 +1,4 @@
+dist/*
+node_modules/*
+yarn-error.log
+.yarn
```

Comment:
Could we just have a universal one?

Reply:
Maybe one for @kenoir - added as separate as the other lambdas in repo have their own .gitignore. Happy to move or leave

Reply:
See other comment about this being non-blocking.